.github/workflows/ci.yaml

```diff
@@ -46,26 +46,36 @@ jobs:
         tanzu plugin install builder --local ${TANZU_HOME}/admin-plugins
         tanzu plugin install test --local ${TANZU_HOME}/admin-plugins
         tanzu plugin list
-    - name: Test
-      run: make test
     - name: Scan Inclusive Terminology
       uses: get-woke/woke-action@v0
       with: 
         fail-on-error: true
         woke-args: -c https://via.vmw.com/its-woke-rules
+    - name: Test
+      run: make test
     - name: Build Version
       run: echo "BUILD_VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
       if: startsWith(github.ref, 'refs/tags/')
     - name: Build
-      run: |
-        make publish
+      run: make build
+    - name: Publish
+      run: make publish
     - uses: actions/upload-artifact@v3
       with:
         name: tanzu-apps-plugin.tar.gz
         path: tanzu-apps-plugin*.tar.gz
         retention-days: 1
     - name: Install
-      run: make install
+      run: |
+        if [[ ${BUILD_VERSION} ]]; then
+          echo $BUILD_VERSION
+          tanzu plugin install apps --version $BUILD_VERSION --local ./artifacts/published/linux-amd64
```

Comment:
you can leave the command out of the `if` as it is the same fot the two branches (`if`/`else`)

Reply:
Updated the code to assign build version only if its not set, and tanzu plugin command moved outside